src/game_objects/player.cpp

```diff
@@ -139,3 +139,15 @@ void Player::reset_position() {
     transform.look_at(direction);
 }
 
+void Player::setup_player_model(std::string &model_name) {
```

Comment:
have setup_model(string) be for everything, but just have default values (like "chest_good" for chest, "boy_two" for player, etc.)